src/Zicht/Bundle/HtmldevBundle/Resources/views/styleguide/design-elements/icons.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends 'ZichtHtmldevBundle:styleguide:component.html.twig' %}
+{% import 'ZichtHtmldevBundle:macros:styleguide.html.twig' as styleguide %}
+
+
+{% block component_list %}
+    <div>
```

Comment:
For some reason this feels redundant. Because there is no styling on this div. So now it only has a padding and some marging and is a block element. But in the other files you add a grid. Don't you need to add the grid here also?

Reply:
I'm not familiar with the grid. I just copy-pasted it form a project :sweat_smile: 

It's also in the Cookiecutter like this.

https://github.com/zicht/cookiecutter-symfony-cms/blob/a54ee99149191ece1ca9567cfe29b3453a169b2b/%7B%7Bcookiecutter.domain_name%7D%7D/htmldev/pages/design-elements/icons.html.twig#L13-L17

My goal is to extract the pages from the project. Not to improve the pages' HTML/CSS content. That could be another PR.

I could add it here though if you have any suggestions.

Reply:
No, that's good enough for me. I only saw with other files that were kind of similar within this PR that it differs. Hence the question